server/api/cart.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express')
+const router = express.Router()
+const { Cart, CartItem } = require('../db')
+
+router.post('/item', async (req, res) => {
```

Comment:
Feels like these two POST routes could be DRYed with some logic

Reply:
Probably so. Might be able to make it call the DB fewer times, but might obscure the 'flow' of this route, though, too. I don't have any great ideas.